sdk/communication/Azure.Communication.Identity/src/autorest.md

```diff
@@ -6,9 +6,9 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 > see https://aka.ms/autorest
 
 ``` yaml
-tag: package-2023-10
+tag: package-2025-04-01-preview
 require:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5797d78f04cd8ca773be82d2c99a3294009b3f0a/specification/communication/data-plane/Identity/readme.md
+    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c7c2694ee53f22dfe56ee0d0fb9875ae5778d0f9/specification/communication/data-plane/Identity/readme.md
```

Comment:
just a reminder, once the PR for rest api get merged we have to update to have the main branch commit